src/Persist.jl

```diff
@@ -798,4 +798,17 @@ function readmgr(jobname::AbstractString)
     mgr::JobManager
 end
 
+"Fetch job result"
+function fetch(jobname::AbstractString)
+    resultfile = joinpath(jobdirname(jobname), resultfilename(jobname))
```

Comment:
Thanks! I'm going to close this. I prefer a better documentation about `readmgr` to changing the API.

Best